src/pool.c

```diff
@@ -3463,7 +3463,11 @@ miner_on_submit(json_object *message, client_t *client)
     if (BN_cmp(hd, bd) >= 0)
     {
         /* Yay! Mined a block so submit to network */
-        log_info("+++ MINED A BLOCK +++");
+        log_info("+++ MINED A BLOCK +++ %.5s %"PRIu64" / %"PRIu64 " %"PRIu64,
```

Comment:
`%0.5s` won't be long enough for a pool with lots of miners (you'll likely get clashes). I suggest `%0.12s`.

Reply:
There can be lots of wallet addresses where the first 5 chars are the same. So if you have thousands of different miners, you could easily have matching first 5 chars for different addresses.